Makefile

```diff
@@ -92,8 +96,13 @@ install:
 	install -m 644 $(wildcard docs/*.8.md) $(PREFIX)/share/man/man8
 	install -d $(PREFIX)/share/man/man5
 	install -m 644 $(wildcard docs/*.5.md) $(PREFIX)/share/man/man5
+	install -D -m 644 ocid.service $(PREFIX)/lib/systemd/system
+	install -D -m 644 ocid.conf $(DESTDIR)/etc
 
 uninstall:
+	systemctl stop ocid.service
+	systemctl disable ocid.service
```

Comment:
Removing a service without disabling it is pretty bad.  I am not crazy about the uninstall path anyways since it just pulls the rug out from under the running apps, without actually stopping them.  But if people agree to remove this, I am fine with it